lib/optimizely/decision_service.rb

```diff
@@ -289,11 +293,11 @@ def get_forced_variation_id(experiment_key, user_id)
       #
       # Returns variation ID into which user_id is forced (nil if no variation)
 
-      forced_variations = @config.get_forced_variations(experiment_key)
+      forced_variations = @config.get_forced_variation(experiment_key, user_id)
```

Comment:
this method should be removed. getforcedvariation from decision service, I assume it should be renamed as getwhitelisted variation which is below.